readium/shared/src/main/java/org/readium/r2/shared/util/downloads/android/AndroidDownloadManager.kt

```diff
@@ -98,6 +93,11 @@ public class AndroidDownloadManager internal constructor(
     private val listeners: MutableMap<DownloadManager.RequestId, MutableList<DownloadManager.Listener>> =
         mutableMapOf()
 
+    /*
+     * If the process is killed at a bad time, listeners for a given [requestId] can be called
```

Comment:
Should we mention this in the `DownloadManager` interface? Otherwise, it could be easily overlooked.

Reply:
I don't remember the point of saying this. It looks like a clarifying note to myself. I think the contract is now quite clear: while the download is not removed, every listener for a request (initially present or added later) will be called once (excepted for progress of course). Not sure if and where we should explain that.